cmd/minikube/cmd/start.go

```diff
@@ -1541,6 +1541,11 @@ func isBaseImageApplicable(drv string) bool {
 
 func getKubernetesVersion(old *config.ClusterConfig) string {
 	if viper.GetBool(noKubernetes) {
+		// Exit if --kubernetes-version is specified.
+		if viper.GetString(kubernetesVersion) != "" {
+			exit.Message(reason.Usage, "cannot specify --kubernetes-version with --no-kubernetes")
```

Comment:
this seems to only handle it the case when it is specified in the Global Config
```
$ mk start --no-kubernetes
😄  minikube v1.24.0 on Darwin 11.6 (arm64)

❌  Exiting due to MK_USAGE: cannot specify --kubernetes-version with --no-kubernetes
```

Please update the Exit with the command that user has to run 
which is "minikube config unset kubernetes-version"
consider this:
```

exit.Message(reason.Usage, "cannot specify --kubernetes-version with --no-kubernetes 
to unset a global config run:

   $ config minikube config unset kubernetes-version
")
```


but if there is an existing cluster running it is not doing the right thing (It is okay if we do it as a follow up PR) See bellow:

```

$ mk start --kubernetes-version=v1.22.1
😄  minikube v1.24.0 on Darwin 11.6 (arm64)
✨  Automatically selected the docker driver
👍  Starting control plane node minikube in cluster minikube
🚜  Pulling base image ...
🔥  Creating docker container (CPUs=2, Memory=1988MB) ...
    > kubectl.sha256: 64 B / 64 B [--------------------------] 100.00% ? p/s 0s
    > kubelet.sha256: 64 B / 64 B [--------------------------] 100.00% ? p/s 0s
    > kubeadm.sha256: 64 B / 64 B [--------------------------] 100.00% ? p/s 0s
    > kubectl: 41.44 MiB / 41.44 MiB [-------------] 100.00% 19.69 MiB p/s 2.3s
    > kubeadm: 40.50 MiB / 40.50 MiB [-------------] 100.00% 15.08 MiB p/s 2.9s
    > kubelet: 137.29 MiB / 137.29 MiB [-----------] 100.00% 20.66 MiB p/s 6.8s

    ▪ Generating certificates and keys ...
    ▪ Booting up control plane ...
    ▪ Configuring RBAC rules ...
🔎  Verifying Kubernetes components...
    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
🌟  Enabled addons: storage-provisioner, default-storageclass
🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
12:35:31 medya/workspace/minikube
go5 ✓
$ mk start --no-kubernetes
😄  minikube v1.24.0 on Darwin 11.6 (arm64)
✨  Using the docker driver based on existing profile
👍  Starting minikube without Kubernetes minikube in cluster minikube
🚜  Pulling base image ...
🏃  Updating the running docker "minikube" container ...
🏄  Done! minikube is ready without Kubernetes!
╭───────────────────────────────────────────────────────────────────────────────────────╮
│                                                                                       │
│                       💡  Things to try without Kubernetes ...                        │
│                                                                                       │
│    - "minikube ssh" to SSH into minikube's node.                                      │
│    - "minikube docker-env" to point your docker-cli to the docker inside minikube.    │
│    - "minikube image" to build images without docker.                                 │
│                                                                                       │
╰───────────────────────────────────────────────────────────────────────────────────────╯

```

Reply:
Yes, have updated the exit message to show config unset command.

Yes, will address the second issue in #12989